packages/gatsby-plugin-google-tagmanager/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
```

Comment:
Hmmm yeah you're right. That is what we should do. We do after all control the builds so this would be easy to setup. We'd just ship the source code and then users could configure their browserlist according to the needs of their project https://github.com/ai/browserslist